### DISCUSSION TITLE
Error opening file msvcp71.dll

### DISCUSSION THREAD

**Discussion by PercyDan54** [itsmattkc/dotnet9x#30] [Q&A] [answered]
I had this error when installing

![image](https://github.com/itsmattkc/dotnet9x/assets/50285552/209878ef-b630-426c-9c64-37c8b17892e6)

Tried installing the os again but didn't fix it, anything I could be doing wrong?

**Comment by itsmattkc** [comment, marked as answer]
This error can be safely ignored. All it means is that file already exists, which isn't a problem. I haven't yet figured out how to make NSIS not throw an error in this scenario.
